src/OneDriveSdk.Common/Authentication/DiscoveryService.cs

```diff
@@ -1,14 +1,6 @@
-// ------------------------------------------------------------------------------
-//  **NOTE** This code was generated by a tool and will occasionally be
-//  overwritten. We welcome comments and issues regarding this code; they will be
-//  addressed in the generation tool. If you wish to submit pull requests, please
-//  do so for the templates in that tool.
-//
-//  This code was generated by Vipr (https://github.com/microsoft/vipr) using
-//  the T4TemplateWriter (https://github.com/msopentech/vipr-t4templatewriter).
-//
+﻿// ------------------------------------------------------------------------------
```

Comment:
Why does it think that this file used to be DeltaResponse? Does DeltaResponse still exist?